docs/src/languages/java/migration.md

```diff
@@ -0,0 +1,178 @@
+---
+title: "Moving a Java application to Platform.sh"
+weight: 2
+sidebarTitle: "Moving to Platform.sh"
+---
+
+
+
+It is widespread to have a Java application that wants to migrate to Platform.sh. It has support to several styles of Java application such as monolith, microservices, stateful, and stateless.
+
+
+
+## Minimum Requirement
+
+
+
+To run a Java application at Platform.sh you must check: 
+
+
+
+* [A supported Java version](https://docs.platform.sh/languages/java.html#supported-versions)
+* [A build management](https://docs.platform.sh/languages/java.html#support-build-automation)
+  * [Gradle](https://docs.gradle.org/current/userguide/gradle_wrapper.html)
+  * [Maven](https://maven.apache.org/) 
+  * [Maven Wrapper](https://www.baeldung.com/maven-wrapper)
+  * [Ant](https://ant.apache.org/)
+* A Git Repository:
+  * [GitHub](https://docs.platform.sh/integrations/source/github.html)
+  * [BitBucket](https://docs.platform.sh/integrations/source/bitbucket.html)
+  * [GitLab](https://docs.platform.sh/integrations/source/gitlab.html)
+  * The default Git repository provided by Platform.sh
+
+{{< note >}}
+A container application cannot be bigger than **8 GB**, checkout, [tunning](https://docs.platform.sh/languages/java/tuning.html) to more details.
```

Comment:
This should be an internal link, using relref.

Also, "see [link]", not "checkout, [link]".

Reply:
Thank you